cfgov/privacy/forms.py

```diff
@@ -132,37 +132,31 @@ def limit_number_of_files(self, files):
     def clean(self):
         super().clean()
         self.require_address_if_mailing()
-        uploaded_files = self.files.getlist('supporting_documentation')
-        self.limit_file_size(uploaded_files)
-        self.limit_number_of_files(uploaded_files)
+        self.uploaded_files = self.files.getlist('supporting_documentation')
+        self.limit_file_size(self.uploaded_files)
+        self.limit_number_of_files(self.uploaded_files)
         return self.escaped_fields()
 
     # Email message
-    def email_body(self, data):
-        num_files = len(data['uploaded_files'])
-        data.update({'num_files': num_files})
+    def email_body(self):
+        data = self.cleaned_data
+        data.update({
+            'num_files': len(self.uploaded_files),
+            'uploaded_files': self.uploaded_files,
+        })
         return loader.render_to_string(self.email_template, data)
 
     def send_email(self):
-        uploaded_files = self.files.getlist('supporting_documentation')
-        data = self.cleaned_data
-        data.update({'uploaded_files': uploaded_files})
-        subject = self.format_subject(data['requestor_name'])
-        from_email = settings.DEFAULT_FROM_EMAIL
-        recipient_list = ['FOIA@consumerfinance.gov']
-
-        body = self.email_body(data)
-
         email = EmailMessage(
-            subject,
-            body,
-            from_email,
-            recipient_list,
-            reply_to=[data['requestor_email']]
+            subject=self.format_subject(),
+            body=self.email_body(),
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            to=['FOIA@consumerfinance.gov'],
```

Comment:
It occurs to me that any cf.gov environment running using settings.production - which I believe includes the dev servers and the UCP Docker stack - will send any emails to this address. Is that a risk / would we potentially want to make this address configurable to avoid any test emails going to the FOIA inbox?

Reply:
ooh, great idea. I'll update that.

Reply:
I made a change here, along with one in the settings repo. Is this what you were thinking of?

Reply:
Yes, thanks!